### PR TITLE
Auto rp

### PR DESCRIPTION
**Pull request by mcardosos**
PTAL @jhendrixMSFT @marstr @vladbarosan 

Thanks you for your contribution to the Azure-SDK-for-Go! We will triage and review it as quickly as we can.

As part of your submission, please make sure that you can make the following assertions:

 - [x] I'm not making changes to Auto-Generated files which will just get erased next time there's a release.
   - If that's what you want to do, consider making a contribution here: https://github.com/Azure/autorest.go
 - [x] I've tested my changes, adding unit tests where applicable.
 - [x] I've added Apache 2.0 Headers to the top of any new source files.
 - [x] I'm submitting this PR to the `dev` branch, or I'm fixing a bug that warrants its own release and I'm targeting the `master` branch.
 - [x] If I'm targeting the `master` branch, I've also added a note to [CHANGELOG.md](https://github.com/Azure/azure-sdk-for-go/blob/master/README.md).
 - [x] I've mentioned any relevant open issues in this PR, making clear the context for the contribution.

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package: github.com/Azure/azure-sdk-for-go
 import:
 - package: github.com/Azure/go-autorest
-  version: ~8.4.0
+  version: ^9.1.0
```
Comment (marstr) [inline glide.yaml]: Let's keep this as `~9.1.0`

Comment (marstr) [reply glide.yaml]: Oh also, in the changelog you mention that you need 9.1.1 not 9.1.0, which is right?

Comment (mcardosos) [reply glide.yaml]: Fixed

Comment (mcardosos) [reply glide.yaml]: well, it is at least 9.1.0, will fix

Comment (mcardosos) [reply glide.yaml]: 9.1.1 is for generator tests

Comment (mcardosos) [reply glide.yaml]: Fixed

Comment (jhendrixMSFT) [reply glide.yaml]: I would be interested to hear from both sides on the motivations behind the change.

Comment (mcardosos) [reply glide.yaml]: Okok, looked for the [difference](https://github.com/Masterminds/glide/blob/master/docs/versions.md)
Now that I understand it, `^` would be better I think

Comment (marstr) [reply glide.yaml]: Agreed, caret seems to do what I thought tilde did. :)

**Comment by marstr** [comment]
Once `glide.yaml` is how you like it, feel free to merge and tag as a release.

**Comment by marstr** [comment]
This PR fixes #688

**Pull request merged**